constants/wram_constants.asm

```diff
@@ -324,3 +324,16 @@ DEF DAYCARELADY_ACTIVE_F         EQU 7
 	const UNLOCKED_UNOWNS_S_TO_W_F
 	const UNLOCKED_UNOWNS_X_TO_Z_F
 DEF NUM_UNLOCKED_UNOWN_SETS EQU const_value
+
+; hVBlank::
```

Comment:
I wouldn't in this PR, but similarly how `macros/wram.asm` was renamed to `macros/ram.asm` we should probably rename this file to `constants/ram_constants.asm` if its going to hold constants used by hram.

Reply:
To do in a subsequent PR!